src/fairseq2/models/wav2vec2/masker.py

```diff
@@ -33,6 +33,7 @@ def __init__(
         max_temporal_mask_prob: float = 0.65,
         spatial_span_len: int = 10,
         max_spatial_mask_prob: float = 0.0,
+        min_num_spans: int = 2,
```

Comment:
Should we have this separate for temporal and spatial? Like `min_num_temporal_spans`, `min_num_spatial_spans`?